### PR TITLE
slight basemodui cleanup & russian translation update

### PR DESCRIPTION
**Pull request by Blueberryy**

### CODE REVIEW

File: resource/basemodui_german.txt [hunk 1]
```diff
@@ -2007,10 +2007,10 @@
...
 "L4D360UI_Controller_SwapJoysticks"		"Ändert die Belegung der Controller-Sticks in UMSEHEN (links) und BEWEGEN (rechts)."
 "[english]L4D360UI_Controller_SwapJoysticks"		"Swap the controller sticks so that LOOK is on the left and MOVE on the right."
-"L4D360UI_Loading_Starring"		"Hauptrollen: "
-"[english]L4D360UI_Loading_Starring"		"Commanders: "
-"L4D360UI_Loading_Costarring"		"Nebenrollen: "
-"[english]L4D360UI_Loading_Costarring"		"Commanders: "
+		"L4D360UI_Loading_Starring"		"Commanders: "
+		"[english]L4D360UI_Loading_Starring"		"Commanders: "
+		"L4D360UI_Loading_Costarring"		"Commanders: "
+		"[english]L4D360UI_Loading_Costarring"		"Commanders: "
```
Comment (BenLubar) [inline resource/basemodui_german.txt:2013]: Was this intentional?

Comment (Blueberryy) [reply resource/basemodui_german.txt:2013]: Yeah. If you go to google translate you will see this text as "Co-starring:". That text was ported from Left 4 Dead 2 to Alien Swarm and they forgot that this one is custom for Alien Swarm

Comment (DukeDukeEm) [reply resource/basemodui_german.txt:2013]: I need to find a translation for "Commander" and "Commanders".
As far as I know, when sombody mentions commander, the person speaking is talking to the player.
But in this case, on the loading screen, I'm not sure who "Commanders" should be. And I also don't know why they are starring. I never really played L4D, so I don't know what all of this means.

Comment (DukeDukeEm) [reply resource/basemodui_german.txt:2013]: Ported here: https://github.com/ReactiveDrop/reactivedrop_translations/discussions/202

**Review by DukeDukeEm — COMMENTED** [review]
Related question to this, move here: https://github.com/ReactiveDrop/reactivedrop_translations/discussions/202

**Pull request merged**